### PR TITLE
Convert stake address tests to hedgehog

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: cardano-cli/test/Test/CLI/Shelley/Golden/StakeAddress/RegistrationCertificate.hs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Test.CLI.Shelley.Golden.StakeAddress.RegistrationCertificate
+  ( golden_shelleyStakeAddressRegistrationCertificate
+  ) where
+
+import Cardano.Prelude hiding (to)
```
Comment (erikd) [inline cardano-cli/test/Test/CLI/Shelley/Golden/StakeAddress/RegistrationCertificate.hs:7]: Wat? I do not see a clashing or alternative defintion.

**Comment by newhoggy** [comment]
> Just remove any un-needed import `hiding` bits and this is good to go.

Done!

File: cardano-cli/test/Test/CLI/Shelley/Golden/Address/Info.hs [hunk 2]
```diff
@@ -4,7 +4,7 @@ module Test.CLI.Shelley.Golden.Address.Info
   ( golden_shelleyAddressInfo
   ) where
 
-import Cardano.Prelude hiding (to)
+import Cardano.Prelude
```
Comment (erikd) [inline cardano-cli/test/Test/CLI/Shelley/Golden/Address/Info.hs:7]: And that should be quashed into the previous commit that added them :D

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**